src/A1_csv_to_json.py

```diff
@@ -339,10 +340,112 @@ def create_json_from_csv(input_directory, filename, parameters, storage=False):
             "No %s" % filename + " assets are added because all "
             "columns of the csv file are empty."
         )
-
-    for column in df:
+    df_copy = df
+    for column in df_copy:
         if column != "unit":
             column_dict = {}
+            # the storage columns are checked for the right parameters,
+            # Nan values that are not needed are deleted
+            if storage == True:
+                # check if all three columns are available
+                if len(df_copy.columns) < 4 or len(df_copy.columns) > 4:
+                    logging.error(
+                        "The file " + filename + ".csv requires "
```

Comment:
```suggestion
                        f"The file {filename}.csv requires "
```

Reply:
In general, could you avoid using "%s" syntax as will be deprecated sooner than the new syntax with f"{filename}" or "{}".format(filename)?

Reply:
ok

Reply:
That being said, do not make more changes now to the `A1_csv_to_json.py` file, to avoid conflicts with the PR #141

Reply:
ok, good that you say that ;)